ocs_ci/ocs/constants.py

```diff
@@ -59,6 +59,7 @@
 # encoded value of 'admin'
 ADMIN_USER = 'admin'
 GB = 1024 ** 3
+SECONDS = 3600
```

Comment:
I wouldn't put this into `constants.py` file as the value is going to be used only on one place. It is also not clear from the variable what is its purpose (seconds of what?).

Reply:
Agreed, will update in next commit. Will remove this and use directly where required.